### PR TITLE
Crafting tutorial

### PR DESCRIPTION
**Pull request by TristanCacqueray**
This change adds a new crafting tutorial.

### CODE REVIEW

File: test/integration/Main.hs [hunk 1]
```diff
@@ -106,8 +106,8 @@ testScenarioSolution _em =
     "Test scenario solutions"
     [ testGroup
         "Tutorial"
-        [ testSolution "move" Default "data/scenarios/02Tutorial/00-move.yaml"
-        , testSolution "turn" Default "data/scenarios/02Tutorial/01-turn.yaml"
+        [ testSolution "move" Default "data/scenarios/02Tutorials/00-move.yaml"
+        , testSolution "turn" Default "data/scenarios/02Tutorials/01-turn.yaml"
```
Comment (byorgey) [inline test/integration/Main.hs:110]: We should also add the new tutorial challenge to the list of tests, I think.

**Review by byorgey — APPROVED** [review]
Looks great to me!

As a reminder, you're welcome/encouraged to open PRs from a branch in the swarm repo instead of from a fork - it makes the restyled bot work better and makes collaboration easier as well.

**Comment by TristanCacqueray** [comment]
Oh right, I'll push the next branch to the origin.

**Comment by byorgey** [comment]
Also, @TristanCacqueray , nice to see you back. :smile:

**Review by xsebek — APPROVED** [review]
Nice tutorial @TristanCacqueray, it's great having more people workshop it! :grinning: 

I wonder if it would be better to introduce `grab` before `make`, but this also makes sense and can definitely be merged as-is.

**Comment by TristanCacqueray** [comment]
I'd be happy to change the order, I figured this one was the easiest.

**Pull request merged**